testing/gcin/gcin.post-install

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+echo -n "updating gtk immodules... "
+[ -x /usr/bin/gtk-query-immodules-2.0 ] && /usr/bin/gtk-query-immodules-2.0 > /etc/gtk-2.0/gtk.immodules
+[ -x /usr/bin/gtk-query-immodules-3.0 ] && /usr/bin/gtk-query-immodules-3.0 > /usr/lib/gtk-3.0/3.0.0/immodules.cache
```

Comment:
This is suspicious, but I don’t know anything about GTK.

@ncopa, could you please look at this PR?